Mathlib/CategoryTheory/Bicategory/Coherence.lean

```diff
@@ -56,23 +56,23 @@ namespace FreeBicategory
 
 variable {B : Type u} [Quiver.{v + 1} B]
 
-/- warning: category_theory.free_bicategory.inclusion_path_aux -> CategoryTheory.FreeBicategory.inclusionPathAux is a dubious translation:
-lean 3 declaration is
-  forall {B : Type.{u2}} [_inst_1 : Quiver.{succ u1, u2} B] {a : B} {b : B}, (Quiver.Path.{succ u1, u2} B _inst_1 a b) -> (CategoryTheory.FreeBicategory.Hom.{u1, u2} B _inst_1 a b)
-but is expected to have type
-  forall {B : Type.{u1}} [_inst_1 : Quiver.{succ u2, u1} B] {a : B} {b : B}, (Quiver.Path.{succ u2, u1} B _inst_1 a b) -> (CategoryTheory.FreeBicategory.Hom.{u2, u1} B _inst_1 a b)
-Case conversion may be inaccurate. Consider using '#align category_theory.free_bicategory.inclusion_path_aux CategoryTheory.FreeBicategory.inclusionPathAuxₓ'. -/
-/-- Auxiliary definition for `inclusion_path`. -/
+/-- Auxiliary definition for `inclusionPath`. -/
 @[simp]
 def inclusionPathAux {a : B} : ∀ {b : B}, Path a b → Hom a b
   | _, nil => Hom.id a
   | _, cons p f => (inclusionPathAux p).comp (Hom.of f)
 #align category_theory.free_bicategory.inclusion_path_aux CategoryTheory.FreeBicategory.inclusionPathAux
 
+/- Porting note: Since the following instance is removed when porting
+`CategoryTheory.Bicategory.Free`, we add it locally here. -/
+/-- Category structure on `Hom a b`. -/
+local instance homCategory' (a b : B) : Category (Hom a b) :=
+  homCategory a b
```

Comment:
This doesn't seem right! There is a category instance on `a ⟶ b` when `a b : FreeBicategory B` already, and that is the one we should be using in this file. I'd prefer to avoid abusing the `FreeBicategory B = B` defeq if we can.

Reply:
But I guess it is okay locally, and if we add a note explaining that what's going on.

Reply:
Certainly the abuse of defeq made debugging difficult. I add a comment explaining why `Hom a b` is used in this file for now.